miri/datamodels/tests/test_reset_switch_charge_decay_model.py

```diff
@@ -122,11 +85,11 @@ def test_fitsio(self):
             self.dataproduct.save(self.testfile, overwrite=True)
             with MiriResetSwitchChargeDecayModel(self.testfile) as readback:
                 self.assertIsNotNone(readback.rscd_table)
```

Comment:
I have tried out this branch with my local B7.5 based MIRICLE build, and for this line 87, it throws the following error
E               AttributeError: No attribute 'rscd_table'

Looking at the latest JWST rcsd schema file at:
https://github.com/spacetelescope/jwst/blob/master/jwst/datamodels/schemas/rscd.schema.yaml
it appears that they have now defined properties such as rscd_group_skip_table, rscd_gen_table, rscd_int1_table, etc.

Should we update the test to assert for one of those instead?